### ISSUE TITLE
VimWiki default bindings work in NeoVim-QT, but not in console Neovim

### ISSUE THREAD

**Issue by cpkio** [neovim/neovim#11999]
- `nvim --version`: **NVIM v0.5.0-415-gf8f41d088**
- `vim -u DEFAULTS` (version: ) behaves differently? **no**
- Operating system/version: **Windows 10 x64**
- Terminal name/version: **any**, confirmed in ConEmu and running `nvim.exe` from windows explorer.
- `$TERM`: **not defined**

### Steps to reproduce using `nvim -u NORC`

```
cd %NEOVIMDIR%\share\nvim\runtime\pack\default\start 
git clone https://github.com/vimwiki/vimwiki.git
nvim -u NORC
> open your vimwiki index.wiki
```

### Actual behaviour
`Backspace` key should bring you to upper level of vimwiki hierarchy (works in Vim and Neovim-QT)

### Expected behaviour
Pressing `Backspace` does not produce the expected behaviour

The Neovim-QT works fine, but console Neovim does not work as expected.

**Comment by Sh3Rm4n** [comment]
Maybe this is caused by conemu itself (or cmd.exe, which I assume you use in conemu). Can you use Backspace in ConEmu in neovim or outside it at all?

**Comment by cpkio** [comment]
For now it works in NeoVim-QT only. Running `nvim.exe` in `cmd`, `powershell` or ConEmu (separately — `powershell` and `cmd` launched from windows run command, so ConEmu does not affect anything) makes no effect. Anyway, if Vim works with any of those mentioned, and NeoVim doesnt, where's the bug supposed to be?

I've tested this with newest NeoVim build, same result — console nvim.exe somewhy doesn't catch up with Backspace bindings, but NeoVim-QT does.

**Comment by Sh3Rm4n** [comment]
I asked, because terminal vim / neovim recognizes key bindings over escape sequences (at least on linux) and the handling of these could differ between terminal emulators. 

However, to make this bug a bit more reproducible and to narrow the scope, did you try by starting neovim, going into insert mode and pressing Control+V and then Backspace? This should print `<BS>`.

**Comment by cpkio** [comment]
`i`,`<c-V>`, `<Bs>` inserts `^H`.

**Comment by cpkio** [comment]
I've found out the funniest thing: `nvim-qt` seems to be producing different behaviour on different systems. At work 0.4.3 release was working fine, but at home at Win10x64 machine `nvim-qt` does not enter cyrillics “Б” letter (I'm a new nvim user and try it only a few days, didnt launch it at home until now). I've done more than half an hour of experiments—terminal `nvim` doesnt even bother to switch languages (with `<c-^>`, not a standard windows layout switching), and `nvim-qt` switches languages with `<c-^>` but enters all the uppercase cyrillic letters but “Б”, inserting “<” instead.

I post it here because it could be connected with keyboard processing routine.

**Comment by erw7** [comment]
The libuv TTY emulation sends the same sequence (`\x08`) on `<C-H>` and `<BS>`. Therefore, `<BS>` mapping cannot be used in the current Windows TUI. Therefore, you need to use the `<C-H>` mapping.

This problem is solved by #11803 in my environment. Perhaps a Windows version with `ENABLE_VIRTUAL_TERMINAL_INPUT` will solve this problem with #11803.

**Comment by erw7** [comment]
It will be closed because #11803 has been merged.
